ChangeLog

```diff
@@ -1,3 +1,56 @@
+===============================================================
+Tag name: noresm3_0_alpha03d
+Date: 21 May 2025
+One-line Summary: Update CTSM and CAM bugfix
+
+Details:
+ - Update CTSM tag to ctsm5.3.034_noresm_v7
+   Removed unnecessary log messages
+ - Update CAM to noresm3_0_002_cam6_4_085
+   Leung_2023 dust scheme
+   Updated Oslo Aero shadow files (src_cam) plus aero_model.F90 to account
+   for modifications in ESCOMP/CAM between cam6_4_070 and cam6_4_085.
+   Move PUMAS external from ESCOMP to NorESMhub
+   Derive new geopotential fields whenever water species are updated
+   Improvements to moving mountains parameterization
+   SE dycore code cleanup
+   Created a list of Oslo Aero aerosol constituent names
+   Added the `ndropmixed` property to Oslo Aero aerosol tracer constituents
+
+List all component tags that were used to create noresm tag:
+ - parallelio : NCAR/ParallelIO             : pio2_6_2
+ - ccs_config : NorESMhub/ccs_config_noresm : ccs_config_noresm0.0.45
+ - cime       : NorESMhub/cime              : cime6.1.73_noresm_v0
+ - share      : NorESMHub/NorESM_share      : share1.1.9_noresm_v0
+ - blom       : NorESMhub/BLOM              : dev1.9.0.13
+ - cam        : NorESMhub/CAM               : noresm3_0_002_cam6_4_085
+ - cdeps      : NorESMhub/CDEPS             : cdeps1.0.70_noresm_v5
+ - cice6      : NorESMhub/NorESM_CICE       : cesm_cice6_5_0_20240702_noresm_v3
+ - cism       : NorESMhub/CISM-wrapper      : cismwrap_2_2_006_noresm_v0
+ - clm        : NorESMhub/CTSM              : ctsm5.3.034_noresm_v6
```

Comment:
This is out of date

Reply:
@gold2718 updates cice and fixed clm tag

Reply:
It looks better now (I checked that all the fxtag statements matches the ChangeLog).